packages/desktop-client/src/components/reports/reports/CustomReport.js

```diff
@@ -109,6 +75,7 @@ export default function CustomReport() {
 
   useEffect(() => {
     async function run() {
+      dispatch(initiallyLoadPayees());
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏I think if yo use `useCachedPayees` hook, then you won't need to manually dispatch this action. Should make the code a bit cleaner.